ModelLibrary/Model/MOrder.cs

```diff
@@ -3414,6 +3441,34 @@ public String CompleteIt()
                 MDocType dt = MDocType.Get(GetCtx(), GetC_DocType_ID());
                 DocSubTypeSO = dt.GetDocSubTypeSO();
 
+                //to check document type if it is pos then we need to check the document type on selected document types
+                if (MDocType.DOCSUBTYPESO_POSOrder.Equals(DocSubTypeSO))
+                {
+                    string paymentbaseType = Util.GetValueOfString(DB.ExecuteScalar("select va009_paymentbasetype from va009_paymentmethod where VA009_PaymentMethod_ID=" + GetVA009_PaymentMethod_ID(), null, Get_Trx()));
```

Comment:
Use upper Case for Keywords and standard notation for query variables

Reply:
Changes Done